Assets/Scripts/Game/PlayerActivate.cs

```diff
@@ -502,70 +503,92 @@ void Update()
                     return;
                 }
 
+                // Play sound when bashing.
+                if (isBash && door.doorType != DoorTypes.DungeonExit)
+                    if (TryGetComponent<DaggerfallAudioSource>(out var dfAudioSource))
+                        dfAudioSource.PlayOneShot(SoundClips.PlayerDoorBash);
+
                 if (door.doorType == DoorTypes.Building && !playerEnterExit.IsPlayerInside)
```

Comment:
Might be cleaner if we just skipped this whole if on a bash.
`door.doorType == DoorTypes.Building && !playerEnterExit.IsPlayerInside && !isBash`
The Discovery part does not seem necessary, you seem to have found how to get the lock level without it